index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube-Clone</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <nav>
+        <div class="left-icons">
+            <a href="" class="menu-icon">
+                <span>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </span>
+            </a>
+            <span>
+                <a hred="">
+                    <img src="Essentials/yt_logo_rgb_light.svg" id="yt-logo">
+                </a>
+            </span>
+        </div>
```

Comment:
Consider using semantic tags for better code readability, user experience, and greater accessibility. Kindly it's better to use a semantic tag like <section> instead of a div.